docs/articles_en/openvino_workflow/model_preparation/Convert_Model_From_TensorFlow.md

```diff
@@ -214,10 +230,16 @@ Model conversion API supports passing TensorFlow/TensorFlow2 models directly fro
      import openvino as ov
 
      model = hub.KerasLayer("https://tfhub.dev/google/imagenet/mobilenet_v1_100_224/classification/5")
-     model.build([None, 224, 224, 3])
```

Comment:
I remember that for some models `build` is required step, otherwise, convert is heavily working or requires `example_input`. So I would suggest to leave this step.

Reply:
Tested on latest master and works without `build()` and does not work after `build()` for some reason.